src/modules/Reviews/ReviewController.ts

```diff
@@ -0,0 +1,28 @@
+// src/controllers/ReviewController.ts
+import express from 'express'
+import ReviewRepository from './interface/ReviewRepository'
+import CreateUserReview from './usecases/CreateUserReview'
+import GetUserReviews from './usecases/GetUserReviews'
+
+class ReviewController {
+  constructor(
+    private reviewRepository: ReviewRepository,
+    private createUserReview: CreateUserReview,
+    private getUserReviews: GetUserReviews,
+  ) {}
```

Comment:
I think if you separate usecase into single small file, we can seperate controller file for each usecase as well. Otherwise, when adding new usecase, we need to change this controller to inject new usecase.

Reply:
Yes, you are right. It was not really apply with open-closed principle. Should consider a better way of how to link in between controller and it's dependent use cases. :D